.github/workflows/test-phpunit.yml

```diff
@@ -98,10 +98,10 @@ jobs:
       - name: Install dependencies
         run: |
           composer install --no-progress --no-suggest --no-interaction --prefer-dist --optimize-autoloader
-          composer remove --dev rector/rector --update-with-dependencies
-          composer remove --dev phpstan/phpstan --update-with-dependencies
-          composer remove --dev codeigniter4/codeigniter4-standard --update-with-dependencies
-          composer remove --dev squizlabs/php_codesniffer --update-with-dependencies
+          composer remove --dev rector/rector
+          composer remove --dev phpstan/phpstan
+          composer remove --dev codeigniter4/codeigniter4-standard
+          composer remove --dev squizlabs/php_codesniffer
```

Comment:
Can we make this a one-liner?

Like `composer remove --dev rector/rector phpstan/phpstan ....`

Reply:
updated.